watertap/property_models/NaCl_property_comparisons.py

```diff
@@ -0,0 +1,568 @@
+import numpy as np
```

Comment:
interesting, but I am not sure that this belongs in the main code base, unless we want to keep it as some sort of example. Either way, I don't think it should here in prop models directory